frontend/src/concepts/connectionTypes/ConnectionTypeForm.tsx

```diff
@@ -111,7 +113,7 @@ const ConnectionTypeForm: React.FC<Props> = ({
                 },
               }
             }
-            onDataChange={setConnectionNameDesc}
+            onDataChange={setConnectionNameDesc ?? (() => undefined)} // onDataChange needs to be truthy to show resource name
```

Comment:
That's a strange requirement from `K8sNameDescriptionField`. Can we update it?

Reply:
Okay it's been updated. I'm not 100% certain on all the use cases for the component, but i think if you want to old preview behavior, they can still pass in the `k8sName.state.immutable = false`